examples/stable-diffusion/README.md

```diff
@@ -31,7 +31,7 @@ python text_to_image_generation.py \
     --model_name_or_path runwayml/stable-diffusion-v1-5 \
     --prompts "An image of a squirrel in Picasso style" \
     --num_images_per_prompt 20 \
```

Comment:
@shepark @regisss If  If we use bs=7, we shouldn't we also set `--num_images_per_prompt` to a multiple of `7`, say `28` so sample runs 4 full batches here, since warmup is 3?

Reply:
Yes, that does make sense.
@shepark You would get better throughput after warmup so I agree with @dsocek that running at least 4 steps should be favored.

Reply:
I am opening a follow up PR on this. 
Will change the `--num_images_per_prompt` in this readme as well.